react/week1/my-app/src/TodoList.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./App.css";
+
+function TodoList({date, description}) {
+    return(
+        <div className="list">
+            <h1>Todo List</h1>
+            <h3>{date}</h3>
+            <p>{description}</p>
+        </div>
+    );
```

Comment:
As mentioned this is a single TodoItem but other than that looks great. Logic wise works as it should!